src/compiler/commandLineParser.ts

```diff
@@ -262,7 +262,9 @@ namespace ts {
                 "es3": ScriptTarget.ES3,
                 "es5": ScriptTarget.ES5,
                 "es6": ScriptTarget.ES6,
+                "es8": ScriptTarget.ES8,

```

Comment:
Can you also add ES2016. it is a bit funny that we skip one ES 2016 (though not very useful but should be there for consistency).